homeassistant/const.py

```diff
@@ -1353,6 +1353,13 @@ class UnitOfPrecipitationDepth(StrEnum):
 CONCENTRATION_PARTS_PER_BILLION: Final = "ppb"
 
 
+class UnitOfBloodGlucoseConcentration(StrEnum):
+    """Blood glucose concentration units."""
+
+    MILLIGRAMS_PER_DECILITER = "mg/dL"
+    MILLIMOLS_PER_LITER = "mmol/L"
```

Comment:
```suggestion
    MILLIMOLE_PER_LITER = "mmol/L"
```
https://www.wikidata.org/wiki/Q55435387